js/client/modules/@arangodb/testutils/result-processing.js

```diff
@@ -1114,6 +1105,37 @@ function getFailedTestCases(options) {
   }
 }
 
+function getGTestResults(fileName, defaultResults) {
+  let results = defaultResults;
+  if (!fs.exists(fileName)) {
+    defaultResults.failed += 1;
+    print(RED + "No testresult file found at: " + fileName + RESET);
+    return defaultResults;
+  }
+  let gTestResults = JSON.parse(fs.read(fileName));
+  results.failed = gTestResults.failures + gTestResults.errors;
+  results.status = (gTestResults.errors === 0) || (gTestResults.failures === 0);
```

Comment:
I don't understand this. It seems that we have both `errors` and `failures`, and that both can be set independently.
Shouldn't it be
```suggestion
  results.status = (gTestResults.errors === 0) && (gTestResults.failures === 0);
```
then?

Reply:
Good catch! These are errors that have been in the code base for who knows how long.. 🙄